content/docs/sidebar.json

```diff
@@ -143,6 +143,10 @@
           "checkpoints"
         ]
       },
+      {
+        "slug": "data-pipelines",
```

Comment:
Should we call them just Pipelines?

Reply:
It's a bit generic. Idk, no strong opinion. Product decision I think, what's the feature called @dberenbaum ? Thanks

Reply:
😅 If it's up to me, I prefer the shorter names for the reasons mentioned in https://github.com/iterative/dvc.org/pull/3830#discussion_r938359041.